cmd/fabric8-jenkins-idler/idler.go

```diff
@@ -52,33 +53,33 @@ func (idler *Idler) Run() {
 	defer cancel()
 
 	// Create Idler controller
-	controller := openshift.NewOpenShiftController(
+	controller := openshift.NewController(
+		ctx,
 		openShift,
 		&tenantClient,
 		idler.features,
 		idler.config,
 		&wg,
-		ctx,
 		cancel,
 	)
 
-	startWorkers(&wg, ctx, cancel, openShift, controller, idler.config.GetDebugMode())
+	startWorkers(ctx, &wg, cancel, openShift, controller, idler.config.GetDebugMode())
```

Comment:
ooi, is there a rule that says the context should come first?

Reply:
Yup